contracts/src/v0.8/llo-feeds/v0.5.0/configuration/Configurator.sol

```diff
@@ -33,23 +35,40 @@ contract Configurator is IConfigurator, ConfirmedOwner, TypeAndVersionInterface,
   error InsufficientSigners(uint256 numSigners, uint256 minSigners);
 
   struct ConfigurationState {
-    // The number of times a new configuration
-    // has been set
+    // The number of times a configuration (either staging or production) has
+    // been set for this DON
     uint64 configCount;
     // The block number of the block the last time
-    /// the configuration was updated.
+    // the configuration was updated.
     uint32 latestConfigBlockNumber;
+    // isFlipped is a bit flip that indicates whether blue is production
+    // exactly one of blue/green must be production at all times.
+    // 0 -> blue is production
+    // 1 -> green is production
+    //
+    // So, to clarify, if isFlipped is false (initial state) then:
+    // [0](blue) is production and [1](green) is staging/retired
+    //
+    // and if isFlipped is true then:
+    // [0](green) is production and [1](blue) is staging/retired
+    //
+    // State is swapped every time a staging config is promoted to production.
+    bool isFlipped;
```

Comment:
not sure if you want to change the name here, but imho `isBlueProduction` or something like that would make so much more sense while reading the code